app/themes/base.js

```diff
@@ -24,32 +24,47 @@ import {
 // and https://github.com/cloudflare/cf-ui/blob/master/packages/cf-style-const/src/variables.js
 
 export default {
-  breakpoints: [
-    '13.6em',
-    '30.4em',
-    '47.2em',
-    '64em',
-    '97.6em'
-  ],
+  breakpoints: {
+    mobile: '30em', // 480px,
+    mobileWide: '37.5em', // 600px,
+    tablet: '48em', // 768px,
+    tabletWide: '56.25em', // 900px,
+    desktop: '64em', // 1024px,
+    desktopWide: '90em', // 1440px,
+    desktopXL: '120em' // 1920px
+  },
   space: [
-    '0.25rem',
-    '0.5rem',
-    '1rem',
-    '2rem',
-    '4rem',
-    '8rem',
-    '16rem',
+    '0rem', // [0] SR: to override default allofthespace with noneofthespace
+    '0.25rem', // [1]
+    '0.5rem', // [2]
+    '1rem', // [3]
+    '2rem', // [4]
+    '4rem', // [5]
+    '8rem', // [6]
+    '12rem', // [7]
+    '16rem', // [8]
+    '20rem', // [9]
+    '24rem' // [10]
   ],
   fontSizes: [
-    '0.875rem',
-    '1rem',
-    '1.25rem',
-    '1.5rem',
-    '2.25rem',
-    '3rem',
-    '5rem',
-    '6rem'
+    '0.8rem', // [0] 10pt
```

Comment:
this is awesome, thanks for finding and sharing!

with this in mind, i'm keen to use the "pentatonic scale" (doubles every fifth step) as described, plus using `em` instead of `rem` when using this scale as recommended in the second article.

```
2^(5/5)      ->  2
2^(4/5)      ->  1.7411
2^(3/5)      ->  1.51572
2^(2/5)      ->  1.31951
2^(1/5)      ->  1.14869
2^(-1/5)     ->  0.87055
2^(-2/5)     ->  0.75785
2^(-3/5)     ->  0.65975
2^(-4/5)     ->  0.57435
2^(-5/5)     ->  0.5
```

so if we follow the second article's advise about using a single up/down scale and nesting `em`s, we might use:

```js
const fontSizes = {
  '-5': 0.50000,
  '-4': 0.57435, 
  '-3': 0.65975,
  '-2': 0.75785,
  '-1': 0.87055,
  0: 1.00000,
  1: 1.14869,
  2: 1.31951,
  3: 1.51572,
  4: 1.74110,
  5: 2.00000,
  // continue with formula:
  //   multiplier ^ (step / interval)
  //   2 ^ (step / 4)
]
```

or more like we did it before:

```js
const fontSizes = {
  0.50000, // [0]
  0.57435, // [1]
  0.65975, // [2]
  0.75785, // [3]
  0.87055, // [4]
  1.00000, // [5]
  1.14869, // [6]
  1.31951, // [7]
  1.51572, // [8]
  1.74110, // [9]
  2.00000, // [10]
  // continue with formula:
  //   multiplier ^ (step / interval)
  //   2 ^ (step / 4)
]
```


happy for this to be a different pull request, will start a conversation about this.